aiida_vasp/workchains/converge.py

```diff
@@ -361,14 +361,20 @@ def _init_pw_context(self):
         self.ctx.converge.pwcutoff_sampling = None
         self.ctx.converge.pw_iteration = 0
         settings.pwcutoff = None
+
         try:
+            if self.inputs.converge.pwcutoff is not None:
+                pwcutoff = self.inputs.converge.pwcutoff
+                settings.pwcutoff = pwcutoff
+
+            # Check inconsistent pwcutoff setting
             parameters_dict = self.inputs.parameters.get_dict()
             electronic = parameters_dict.get('electronic', None)
-            if electronic is not None:
-                pwcutoff = electronic.get('pwcutoff', None)
-                settings.pwcutoff = pwcutoff
+            if (electronic is not None) and ('pwcutoff' in electronic):
+                self.report("value of 'parameters.electronic.pwcutoff' is overrode with that of 'convere.pwcutoff'")
```

Comment:
Fix spelling `converge`. Also we should give an error message that is not referring to parameters, but try to spell out what is happening. Suggestion: `The 'pwcutoff' supplied in the global input parameters was overridden by the 'pwcutoff' supplied to the workchain` or something along those lines. I guess we have not given the `parameters` a name yet, so not sure `global` is the right on. @atztogo @zhubonan @JPchico thoughts?

Reply:
Thank you for your suggestion and correction. For the moment, I have replaced the error message with your suggestion.